isis/scripts/install_isis.sh

```diff
@@ -0,0 +1,472 @@
+    printf "\\n"
+    printf "  - Press ENTER to confirm the ISISDATA location\\n"
+    printf "  - Press CTRL-C to abort the installation\\n"
+    printf "  - Or specify a different location below\\n"
+    printf "\\n"
+    printf "[%s] >>> " "$ISISDATA_PREFIX"
+
+    read -r isis_data_path
+
+    # If no input, use default path
+    if [ -n "$isis_data_path" ]; then
+        ISISDATA_PREFIX=$(eval printf "%s" $isis_data_path)
+    fi
+
+    echo "Setting ISISDATA to $ISISDATA_PREFIX"
+fi
+
+# Verify if the folder exists
+if [ ! -d "$ISISDATA_PREFIX" ]; then
+    if [ "$DOWNLOAD_DATA" = "YES" ]; then
+        echo "Creating folder $ISISDATA_PREFIX"
+        mkdir -p $ISISDATA_PREFIX || failed_command "Creating $ISISDATA_PREFIX"
+    else
+        echo "Not creating directory $ISISDATA_PREFIX since the --no_data flag was set"
+    fi
+fi
+
+$MINIFORGE_DIR/bin/$CLIENT env config vars set -n $ENV_NAME ISISDATA=$ISISDATA_PREFIX ISISROOT=$MINIFORGE_DIR/envs/$ENV_NAME || failed_command "Mamba config var set"
+
+if [[ "$MINIFORGE_DIR" == *"envs/$ENV_NAME"* ]]; then
+    ENV_PATH="$MINIFORGE_DIR"
+else
+    ENV_PATH="$MINIFORGE_DIR/envs/$ENV_NAME"
+fi
+
+DOWNLOAD_ISIS_DATA_SCRIPT="$ENV_PATH/bin/downloadIsisData"
+if [[ ! -f "$ENV_PATH/bin/downloadIsisData" && ! -f "$ENV_PATH/etc/isis/rclone.conf" ]]; then
+    if ! [ -f "$DOWNLOAD_ISIS_DATA_SCRIPT" ]; then
+        printf "\nInstalled download script: $DOWNLOAD_ISIS_DATA_SCRIPT\n"
+        curl  --output "$ENV_PATH/bin/downloadIsisData" -LJO https://github.com/DOI-USGS/ISIS3/raw/dev/isis/scripts/downloadIsisData
+        chmod +x "$ENV_PATH/bin/downloadIsisData"
+    fi
+    if ! [ -f "$ENV_PATH/etc/isis/rclone.conf" ]; then
+        printf "\nInstalled isis rclone config: $ENV_PATH/etc/isis/rclone.conf\n"
+        # Verify if the folder exists
+        if ! [ -d "$ENV_PATH/etc/isis/" ]; then
+            echo "Creating folder $ISISDATA_PREFIX"
+            mkdir -p $ENV_PATH/etc/isis/ || failed_command "Creating $ENV_PATH/etc/isis/"
+        fi
+        curl --output "$ENV_PATH/etc/isis/rclone.conf" -LJO https://github.com/DOI-USGS/ISIS3/raw/dev/isis/config/rclone.conf
+    fi
+else
+    printf "\nFound download script: $DOWNLOAD_ISIS_DATA_SCRIPT\n"
+fi
+
+if [ "$DOWNLOAD_DATA" = "NO" ]; then
+    exit 0
+fi
+
+printf "\n\n"
+echo "ISISDATA is required for most applications."
+echo "This will download several gigabytes of data and may take a few hours."
+echo "ISISDATA path is currently set to $ISISDATA_PREFIX"
+echo "You can do this later, read more at:" 
+printf "\n\thttps://astrogeology.usgs.gov/docs/how-to-guides/environment-setup-and-maintenance/isis-data-area/\n\n"
+echo "Do you want to install base ISISDATA now? This can be done later. [yes|no]"
+ans="no"
+printf "[%s] >>> " "$ans"
+
+read -r ans
+
+# If no input, use default path
+if [ "$ans" = "" ]; then
+    ans="no"
+fi
+
+ans=$(echo "${ans}" | tr '[:lower:]' '[:upper:]')
+while [ "$ans" != "YES" ] && [ "$ans" != "NO" ]
```

Comment:
This `while` seems unnecessary because the script defines `ans=no` by default above.